src/configuration/testcafe-configuration.ts

```diff
@@ -74,8 +74,8 @@ interface TestCafeStartOptions {
 type BrowserInfoSource = BrowserInfo | BrowserConnection;
 
 export default class TestCafeConfiguration extends Configuration {
-    public constructor (configFile = CONFIGURATION_FILENAME) {
-        super(configFile);
+    public constructor (configFiles: string | string[] = Configuration._getFileNamesByPriority(BASE_CONFIGURATION_FILENAME)) {
```

Comment:
I propose moving the code from [here](https://github.com/DevExpress/testcafe/pull/6319#discussion_r661371305) in the  `src/configuration/formats.ts` file and reuse the necessary constants from it.